cmake/ext_ortlib.cmake

```diff
@@ -2,7 +2,9 @@ if(_ONNXRUNTIME_EMBEDDED)
   set(ONNXRUNTIME_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/../include/onnxruntime/core/session)
   set(ONNXRUNTIME_LIB_DIR "")
 else()
-  set(ONNXRUNTIME_VER "1.10.0" CACHE STRING "ONNX Runtime version")
+  if (NOT ONNXRUNTIME_VER)
```

Comment:
Is that what a cache variable doing. If it is not set, it will create one with the specified value?

Reply:
Ah cool. Didn't realize it did that. Will remove this change.